src/docs/processing-tickets.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Processing Issues & PRs"
+---
+
+This document describes Sentry's process for handling issues and PRs—let's use "tickets" to refer to both together, if that's okay—that are created by external contributors in our public GitHub repos.
+
+## 1. Validate
+
+*SLA: 1 hour*
+
+The Open Source Team maintains [a bot](https://github.com/getsentry/.github/blob/main/.github/workflows/validate-new-issue.yml) that validates incoming tickets. Invalid tickets are closed; valid tickets are given the label `Status: Unrouted`. The bot is installed on three repos: `sentry`, `sentry-docs`, and `onpremise`.
```

Comment:
May want to link "Open Source Team" to https://github.com/orgs/getsentry/teams/open-source

Reply:
Looks like that doesn't show anything meaningful to anons. Original had a link to the internal Notion doc, dropped that here for the same reason, though I kept the product and services registry link. Should we link to Notion?

Reply:
Leaving as-is for now.

Reply:
Eh, if we want to be consistent, then we should either assume that the audience is okay not being able to see certain links or remove all links to non-public resources. Keeping one but not adding another one doesn't make sense to me.